pkg/virt-controller/watch/vmi.go

```diff
@@ -709,6 +712,16 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 
 	case vmi.IsRunning():
 		if !vmiPodExists {
+			virtHandlerAlive, err := isVirtHandlerAlive(c.clientset, vmi.Status.NodeName)
+
+			if err != nil {
+				log.Log.Errorf("virt-handler pod is not responsible %v", err)
```

Comment:
That error message sounds weird. IMO it should read `virt-handler pod is not alive ...`. Also it could be a warning instead of an error.

Reply:
I've dropped this message.